src/utils/embeddableUrl.tsx

```diff
@@ -0,0 +1,121 @@
+import {
```

Comment:
Why the rename from `.ts` to `.tsx`?

This utility file doesn't seem to contain any TSX syntax

Reply:
renamed it back to `.ts` 741d64c